crates/modelardb_server/src/main.rs

```diff
@@ -44,7 +44,10 @@ static ALLOC: snmalloc_rs::SnMalloc = snmalloc_rs::SnMalloc;
 
 /// The port of the Apache Arrow Flight Server. If the environment variable is not set, 9999 is used.
 pub static PORT: Lazy<u16> = Lazy::new(|| match env::var("MODELARDBD_PORT") {
-    Ok(port) => port.parse().unwrap(),
+    Ok(port) => port
+        .parse()
+        .map_err(|_| "MODELARDBD_PORT must be between 1 and 65535.")
```

Comment:
1024-49151?

Reply:
I suggest we keep it between 1 and 65535 since we do not restrict the port number the user can use, so if the use has permission from the operating `modelardbd` runs fine on port 1 and 65535.